modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/UpdateAllCommand.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.command;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.ignite.internal.schema.BinaryRow;
+import org.apache.ignite.internal.storage.RowId;
+import org.apache.ignite.internal.util.CollectionUtils;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * The command for update batch of entries.
+ */
+public class UpdateAllCommand extends PartitionCommand {
+    /**
+     * Rows to update.
+     */
+    private transient HashMap<RowId, BinaryRow> rowsToUpdate;
```

Comment:
Why it's HashMap and not Map?

Reply:
It gives additional limitation, Hash map can store a null values.

Reply:
I didn't get you point, never mind though, it's a minor.